### ISSUE TITLE
Error accessing personal information

### ISSUE THREAD

**Issue by trodaway** [the-scouts/compass-interface-core#90]
Exception is raised when trying to access personal information of members within my hierarchy. Nothing raised when accessing my own information though. I believe my role doesn't give me access to my hierarchy's disabilities/qualifications/hobbies ("other sections") information within Compass, so I think this is a case that needs handling to prevent the error.

```python-traceback
Traceback (most recent call last):
  File "<input>", line 3, in <module>
  File "<path_to_project>\venv\lib\site-packages\compass\core\people.py", line 43, in personal
    return self._scraper.get_personal_tab(membership_number)
  File "<path_to_project>\venv\lib\site-packages\compass\core\_scrapers\member.py", line 266, in get_personal_tab
    details["disabilities"] = {k: v for k, _, v in (row[0][0].text.partition(" - ") for row in div_profile_tbl[9][2])}
  File "src\lxml\etree.pyx", line 1161, in lxml.etree._Element.__getitem__
IndexError: list index out of range
```

**Comment by AA-Turner** [comment]
Hi Tim, 

Could you email me with the role you're logging in with at my firstname . surname at scouts.org.uk please?

Thanks,
Adam

**Comment by AA-Turner** [comment]
Closed in #91
